src/components/MovieCard/MovieCard.js

```diff
@@ -64,13 +64,12 @@ const MovieCard = ({ movie }) => {
           <img
             src={movie.poster_path}
             alt="Movie Poster"
-            className="movie-card__image"
             onClick={handleMovieDetails}
           />
         </div>
         <div className="movie-card__information">
-          <div className="movie-card__information-title">{movie.title}</div>
-          <div className="movie-card__information-year">
+          <div className="movie-card__information__title">{movie.title}</div>
```

Comment:
Beware that in the BEM subtask this convention will be incorrect - for now, you can leave it as it is, but I want to stress that point

Reply:
Noted, I will fix it in BEM subtask.